launcher/ui/dialogs/NewInstanceDialog.cpp

```diff
@@ -151,6 +151,9 @@ void NewInstanceDialog::setSuggestedPack(const QString& name, InstanceTask* task
 {
     creationTask.reset(task);
     ui->instNameTextBox->setPlaceholderText(name);
+    ui->instNameTextBox->setText(name);
+    ui->instNameTextBox->selectAll();
+    ui->instNameTextBox->setFocus();
```

Comment:
Can you do it without the setFocus?

Reply:
Detecting already made changes and not resetting the name could also be a thing, along with a reset button that sets the text to the current suggested name and clears that flag.

Reply:
Done all of this, let me know if there is a better way to detect if the user has clicked inside the text box